ShoppingCart-topic1/src/MailList.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+
+public class MailList implements List {

```

Comment:
In real life the mailList should use an email service to send all the notifications to a list of people to be notified